### ISSUE TITLE
Swords transmog bugged on Shamans

### ISSUE THREAD

**Issue by Stormbreaker23** [GameFreedomGG/Sindragosa#277]
Decription: Hello please, can u somehow fix so shamans can transmogs swords? every class including druids can transmog them but shamans cant.. 1h and 2h. 

How it works:

How it should work:

Source (you should point out proofs of your report, please give us some source):

**Comment by UltraNix** [comment]
No. druid cannot transmog with 2-h swords. And shaman also cannot wield 2h swords.
